lib/cn/node_split.c

```diff
@@ -177,6 +180,8 @@ find_inflection_key(
         curr = iter->ks->ks_kblks + iter->offset;
 
         if (curr == inflection_kblk) {
+            log_debug("node %lu inflection kblock (kvset, kblock): (%lu, %u)",
```

Comment:
Do we need a similar log message at the end of this for loop where we also set offsets[i]?

Reply:
Added in the next push.